From 21a17a7caf43236590d2e7319e176cb19c42f9aa Mon Sep 17 00:00:00 2001
From: Vladimir Dvorak <vladimir.dvorak@jetbrains.com>
Date: Sun, 29 Nov 2020 20:08:57 +0100
Subject: [PATCH 18/42] dcevm15 - DON'T clear F2 in CP cache after indy
 unevolving

It's not clear why it was cleared in dcevm7-11
---
 src/hotspot/share/oops/cpCache.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hotspot/share/oops/cpCache.cpp b/src/hotspot/share/oops/cpCache.cpp
index 167cb274661..7e72641ec5a 100644
--- a/src/hotspot/share/oops/cpCache.cpp
+++ b/src/hotspot/share/oops/cpCache.cpp
@@ -654,7 +654,7 @@ void ConstantPoolCacheEntry::clear_entry() {
 
   if (clearData) {
     if (!is_resolved_reference()) {
-      _f2 = 0;
+      // _f2 = 0;
     }
     // FIXME: (DCEVM) we want to clear flags, but parameter size is actually used
     // after we return from the method, before entry is re-initialized. So let's
-- 
2.23.0

